opfunu/cec_based/cec2022.py

```diff
@@ -579,7 +579,7 @@ def __init__(self, ndim=None, bounds=None, f_shift="shift_data_11", f_matrix="M_
         self.xichmas = [20, 20, 30, 30, 20]
         self.lamdas = [1e-26, 10, 1e-6, 10, 5e-4]
         self.bias = [0, 200, 300, 400, 200]
-        self.g0 = operator.expanded_scaffer_f6_func
+        self.g0 = operator.rotated_expanded_schaffer_func
```

Comment:
So https://github.com/P-N-Suganthan/2022-SO-BO/blob/main/CEC2022%20TR.pdf is confusing.
The first reference says: "Shifted and full Rotated Expanded Schaffer’s f6 Function", then the title on page 7 for the 3rd function is "Shifted and full Rotated Expanded Schaffer’s F7" and the same page refers in figure 3 text "Shifted and full Rotated Expanded Schaffer’s f6 Function"

So I went to the python source code for CEC2022 https://github.com/P-N-Suganthan/2022-SO-BO/blob/main/Python-CEC2022.zip extracted the python code there and optimized it for speed and use of numpy. This gives parity with the reference code and the name is based on that since it is supposed to represent Shifted and Rotated Schaffer F6. Happy to change the name if you have a preference for something else.